components/datetime/src/format/datetime.rs

```diff
@@ -167,8 +160,10 @@ where
             input!(year = input.year);
             input!(related_iso = year.related_iso());
 
-            // TODO: This should use a latin FDF
-            try_write_number(w, fdf, related_iso.into(), l)?
+            FixedDecimal::from(related_iso)
```

Comment:
Nit: Add comment that `impl Writeable for FixedDecimal` is what the spec says to use here